src/_locales/es/messages.yml

```diff
@@ -4,8 +4,7 @@ anchorAuthor:
 anchorTranslator:
   description: Translator shown on about tab.
   message: >-
-    <a href="http://greydus.tk" target="_blank" rel="noopener
-    noreferrer">Greydus</a>
+    <a href="http://greydus.tk" target="_blank" rel="noopener noreferrer">Greydus</a>
```

Comment:
This change is unnecessary, the style is generated automatically by YAML tools. The values they represent have no difference. It will get back if we normalize all locale files the next time.

You can try `yarn i18n` if you are familiar with the Node.js tool chain.